src/main/java/de/dennisguse/opentracks/content/provider/CustomContentProvider.java

```diff
@@ -356,11 +362,46 @@ private Uri insertMarker(Uri url, ContentValues contentValues) {
         throw new SQLException("Failed to insert a marker " + url);
     }
 
+    /**
+     * Return the string representing the query that compute sensor stats from trackpoints table.
```

Comment:
* Return the string representing the query that compute sensor stats from trackpoints table.
* It computes the average for heart rate, cadence, and power (duration-based average) and the maximum for heart rate and cadence.